### PR TITLE
Header-SubHeader Added

### PR DESCRIPTION
**Pull request by developerwaleed**
- Only the basic skeleton is made till now.
- Project must be seen in a width of 500px.
- Header Added
- Sub Header added
- Social icons added
- Button with hover function added.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Waleed's Portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <div class="logo">Welcome</div>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please add a link to your logo with an `a tag`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            <img class="menu" src="./icons/menu.png" alt="menu" height="25" width="25">
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please kindly wrap your hamburger button in a `nav tag`.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        </header>
+        
+        <div class="landing-section">
+            
+            <section class="landing-text">
+                
+                <h2>Hey there. I’m <br>Waleed<br>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please your main title should be in an  `h1 tag`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                </h2>
+                <p id="landingtext2">
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.<br>
+                    <button type="button" class="connect">Lets Connect!
+                    </button>
+                </p>     
+                <div class="social-icon">
+                    <img src="./icons/social.png" alt="Social Media">
+                </div>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please your social icons should be wrapped in  `ul tag` to group them and each icon should be in a  `li tag` and have a link with the `a tag`. **Good Example Below**
```
<ul class="social-media-icon">
      <li class="social-media"><a class="list-style" href="#" ><img src="Linkedin-icon.svg"  alt="" ></a></li> 
      <li class="social-media"><a class="list-style" href="#" ><img src="mediaThree.svg"     alt="" ></a></li> 
      <li class="social-media"><a class="list-style" href="#" ><img src="mediaFour.svg"      alt="" ></a></li> 
      <li class="social-media"><a class="list-style" href="#" ><img src="mediaFive.svg"      alt=""></a></li>
</ul>
```

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @caasperr, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+              <img class="menu" src="./icons/menu.png" alt="menu" height="25" width="25">
+              
+            </div>
+        </nav>
+        </header>
+        
+        <div class="landing-section">
+            
+            <section class="landing-text">
```
Comment (aimemalaika) [inline index.html:27]: - [ ] Kindly make the mobile version responsive on the paragraph an the `h1`
- 
Screen Shot 2022-04-21 at 11 59 09

File: index.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+                
+                <h1>Hey there. I’m <br>Waleed<br>
+                <span id="landingtext">I’m a software <br>developer</span>
```
Comment (aimemalaika) [inline index.html]: - [ ] The subtitle (if existing) uses a header tag `hx (examples: `h1`,`h2`,`h3`, etc...)` Kindly check that.

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @caasperr , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Highlights


## Required Changes ♻️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (developerwaleed) [reply index.html]: I didn't understand what you are trying to say can you please elaborate? @aimemalaika

**Review by phelian23 — APPROVED** [review]
Hi @caasperr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**